README.md

```diff
@@ -100,9 +100,10 @@ your project as explained below.
 
 To use Swift with Protocol buffers, you'll need:
 
-* A recent Swift 3 compiler that includes the Swift Package Manager.  The Swift
-protobuf project is being developed and tested against the latest release version of
-Swift available from [Swift.org](https://swift.org)
+* A recent Swift 3 compiler (Xcode 8.x or later).  Support is included for the Swift
```

Comment:
Should we be more specific here?

    (we currently require Swift 3.0.1 or later, as provided by Xcode 8.1 or later)